mealpy.py

```diff
@@ -1,126 +1,111 @@
 import getpass
 import json
-import requests
 import time
 
+import requests
 from apscheduler.schedulers.blocking import BlockingScheduler
 
 BASE_DOMAIN = 'secure.mealpal.com'
-BASE_URL = 'https://' + BASE_DOMAIN
-LOGIN_URL = BASE_URL + '/1/login'
-CITIES_URL = BASE_URL + '/1/functions/getCitiesWithNeighborhoods'
-MENU_URL = BASE_URL + '/api/v1/cities/%s/product_offerings/lunch/menu'
-RESERVATION_URL = BASE_URL + '/api/v2/reservations'
-KITCHEN_URL = BASE_URL + '/1/functions/checkKitchen3'
```

Comment:
I'm happy with the format string changes, but any reason to eliminate them as constants at the top? I know they are only used once, but IMO having them up here would be easier to find / change if the API changes.

Reply:
No reason, this is my personal preference. They're only used in one place but incur a layer of indirection when statically analyzing code. I don't feel strongly, I can move them back to global constants if you press for it.

In practice, API changes are going to require looking at the call sites to make changes to handle the new request params and response. If the API changes under us (maybe endpoints routes change), we won't find out until a function unexpectedly breaks, and then we'll need to dereference the constant.

Reply:
I tend to prefer having as many "hard-coded" strings isolated from the code as constants as possible, and indirection shouldn't be an issue with IDEs.

Reply:
I will move them back to constants.